composer.json

```diff
@@ -23,5 +24,11 @@
             "PicoPluginInterface": "lib/",
             "AbstractPicoPlugin": "lib/"
         }
+    },
+    "scripts": {
+        "post-install-cmd": [
```

Comment:
I suppose that when Pico 1.1.0 is ready to go, a Bootstrap 4 CDN will be available, if not, its just as simple to use Bootstrap 3. Then there will be no need to include with composer, or use the `post-install-cmd` script.